gulpfile.js

```diff
@@ -0,0 +1,24 @@
+const del = require('del');
+const eslint = require('gulp-eslint');
+const gulp = require('gulp');
+const zip = require('gulp-zip');
+
+gulp.task('build', ['lint', 'clean:dist'], function () {
+  return gulp.src('src/*')
+    .pipe(zip('addon.xpi'))
+    .pipe(gulp.dest('dist'));
+});
```

Comment:
I'll do it in a separate PR. We also have the #44 issue which covers generating those RDF files from templates, so I may try and bundle those two tasks into a tidy commit along with an updated README.